src/jit/compiler.cpp

```diff
@@ -5815,16 +5819,52 @@ var_types Compiler::getEightByteType(const SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASS
         }
         else
         {
-            assert(false && "getEightByteType Invalid SSE classification type.");
+            assert(false && "GetEightByteType Invalid SSE classification type.");
         }
         break;
     default:
-        assert(false && "getEightByteType Invalid classification type.");
+        assert(false && "GetEightByteType Invalid classification type.");
         break;
     }
 
     return eightByteType;
 }
+
+//------------------------------------------------------------------------------------------------------
+// GetStructTypeOffset: Gets the type, size and offset of the eightbytes of a struct for System V systems.
```

Comment:
I just moved this routine from codegenlinear.h/codegenxarch.cpp to compiler.h/compiler.cpp verbatim and didn't change function implementation.

Function return is still void, it just that it returns the values through out params, which is indicated in Arguments" section.